schema/config.js

```diff
@@ -7,12 +7,12 @@ module.exports = {
 		properties: {
 			wsPort: {
 				type: 'integer',
-				minimum: 1,
+				minimum: 1024,
 				maximum: 65535
 			},
 			httpPort: {
 				type: 'integer',
-				minimum: 1,
+				minimum: 1024,
```

Comment:
I believe many users in production environment may run the application on `80` or `443`, who are directly serving lisk instance without a HTTP server. The change in this config may cause problem to them. 

 @karmacoma What do you think?

Reply:
@nazarhussain Agreed 👍